Sources/AppDebugMode/Utils/StandardOutputService.swift

```diff
@@ -47,19 +47,43 @@ final class StandardOutputService: ObservableObject {
         didRedirectLogs = true
 
         setvbuf(stdout, nil, _IONBF, 0) // set output as unbuffered
+        // Because all the standard print() logs are stread through STDOOUT
         dup2(pipe.fileHandleForWriting.fileDescriptor, STDOUT_FILENO)
+        pipe.fileHandleForReading.readabilityHandler = { [weak self] handle in
+            let data = handle.availableData
+            let readString = String(data: data, encoding: .utf8) ?? "<\(data.count) bytes of non-UTF-8 data>\n"
+
+            self?.redirectLog(readString)
+        }
 
+        // Because all the OSLog logs are stread through STDErr
+        dup2(pipe.fileHandleForWriting.fileDescriptor, STDERR_FILENO)
         pipe.fileHandleForReading.readabilityHandler = { [weak self] handle in
             let data = handle.availableData
-            let str = String(data: data, encoding: .utf8) ?? "<\(data.count) bytes of non-UTF-8 data>\n"
-
-            DispatchQueue.main.async {
-                let log = Log(message: str)
-                guard !log.message.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty else {
-                    return
-                }
-                self?.capturedOutput.append(log)
+            var readString = String(data: data, encoding: .utf8) ?? "<\(data.count) bytes of non-UTF-8 data>\n"
+
+            // #"^(\nOSLOG-.*?\t)"# Because every OSLog entry starts with newline continues with "OSLog- ..."
+            // and ends with /t on the second line
+            // Finding first match because the rest of the string should stay intact.
+            // Only want to trip the OSLog MetaData
```

Comment:
```suggestion
            // Trim OSLog metadata prefix
```
4 (6 zalomenych) riadkov komentarov na 3 riadky kodu mi pride, ze az prilis vysvetluju, co sa tam deje. 😄  Napadol mi pri tomto ten vyrok

> Don't write comments, rewrite your code